etc/delphix-platform/ansible/apply

```diff
@@ -19,6 +19,7 @@ set -o pipefail
 
 SOURCE_DIRECTORY="$(readlink -f "${BASH_SOURCE%/*}")"
 DLPX_ANSIBLE_DIRECTORY="${DLPX_ANSIBLE_DIRECTORY:-$SOURCE_DIRECTORY}"
+ROOT_FILESYSTEM=$(df / --output=source | tail +2)
```

Comment:
Could we use `zfs list -Ho name  /` instead?

Reply:
Yes, I like that better.